sdk/keyvault/keyvault-keys/review/keyvault-keys.api.md

```diff
@@ -333,6 +343,9 @@ export class LocalCryptographyClient {
     key: JsonWebKey;
     verifyData(algorithm: LocalSupportedAlgorithmName, data: Uint8Array, signature: Uint8Array): Promise<VerifyResult>;
     wrapKey(algorithm: LocalSupportedAlgorithmName, key: Uint8Array): Promise<WrapResult>;
+    unwrapKey(algorithm: LocalSupportedAlgorithmName, encryptedKey: Uint8Array): Promise<UnwrapResult>;
+    decrypt(algorithm: LocalSupportedAlgorithmName, ciphertext: Uint8Array): Promise<DecryptResult>;
+    sign(algorithm: LocalSupportedAlgorithmName, digest: Uint8Array): Promise<SignResult>;
```

Comment:
This is an interesting one. These methods are not available yet on the local cryptography client. We should probably defer adding them this month until we can budget adding proper local support.

Reply:
Sure I can remove them

Reply:
Should I just remove the entire LocalCryptographyClient and instead add an optional constructor param to the CryptographyClient with the JsonWebKey?